packages/editor-ui/src/components/TriggerPanel.vue

```diff
@@ -171,36 +175,34 @@ export default defineComponent({
 
 			return null;
 		},
+		triggerPanel() {
+			return this.nodeType?.triggerPanel;
+		},
```

Comment:
might be simpler if you return the type `TriggerPanelDefinition` here. Then you can simplify the rest of the assertions here.
```suggestion
		triggerPanel(): TriggerPanelDefinition | undefined {
		  const panel = this.nodeType?.triggerPanel;
			if (triggerPanel === 'object' && triggerPanel !== null) { return panel }
			return undefined;
		},
```

Reply:
Makes sense since the `boolean` version is not used anywhere here